src/main/java/com/github/pedbertor/gol/controller/Controller.java

```diff
@@ -46,11 +47,13 @@ public class Controller {
      * @param cellStateProcessor the cell state processor
```

Comment:
@Sumlarys you forgot to update the constructor javadoc for Controller, please check it.